bin/clonetb

```diff
@@ -81,8 +81,7 @@ clone() {
 clone_cv32e40x() {
   CV_CORE=cv32e40x
   VERIF_ENV_REPO=https://github.com/openhwgroup/cv32e40x-dv.git
-  VERIF_ENV_REF=443f11c
-
+  VERIF_ENV_REF=be17b8902002f91803abde4bfb8caa91088575e1
```

Comment:
Maybe this will be fixed in another commit, but the hash is not the updated one.

Reply:
It was the latest when I made the PR.
And the newer commits are seemingly just vplan updates.

> Maybe this will be fixed in another PR

I'd prefer this one.

![bilde](https://github.com/openhwgroup/core-v-verif/assets/75431602/acc1771e-72ce-4e97-9005-4cd2f701ab37)